MBVM-84: Support database-only mirror as an alternative

yvanzo opened the pull request:
# MBVM-84: Support database-only mirror

## Problem

The default configuration assumes that the web stuff should be mirrored as well.

## Solution

Before building images or any other configuration step,
replace the default `docker-compose.yml` with alternative `docker-compose.alt.db-only-mirror.yml` as follows:

```bash
admin/configure with alt-db-only-mirror
```

## Checklist for author
<!--
    The tasks you have to do to get your change ready for review. Use this if
    you draft a pull request. Mark done tasks with an [x] as you progress. See
    https://github.blog/2019-02-14-introducing-draft-pull-requests/
-->

* [x] Test installing a database mirror from scratch
* [x] Update `README.md`

yvanzo committed:
MBVM-84: Database-only mirror alternative

The alternative base compose file is placed into the same directory as
the default base compose file so as not to break relative paths set in
override files such as `compose/replication-token.yml`.

The script `admin/compose` is updated to handle this alternative too.

Note that a Redis instance is currently still required even though it is
not actually used. See the ticket MBS-12440 for follow-up.

Upgrade notes: If you want to switch a running mirror to this base
compose file, you will have to re-add other compose files you had.
Check with `admin/configure show` before and after running the command.

dgzefem commented:
I gave this a try. Everything looks good, I've executed a replication and all worked well I believe. Only thing I had to do (not sure if this is intented), is that I had to re-execute:

```shell
$ ./admin/configure add replication-token
$ ./admin/configure add replication-cron
```

...because my token was wiped after I executed admin/configure with alt-db-only-mirror.

Also minor update, the PR's description is outdated (`db-only-mirror`).

Thanks for this!

yvanzo commented:
> Only thing I had to do (not sure if this is intented), is that I had to re-execute [...] ...because my token was wiped after I executed `admin/configure with alt-db-only-mirror`.

Yes this command is expected to disable other compose files, that is why the instructions are given just after cloning the repository. You can check currently enabled compose files with `admin/configure show`.

Thanks to your feedback, I added a warning against not using it as a first step in `README.md` on [line 106](https://github.com/metabrainz/musicbrainz-docker/pull/226/commits/2cc17c014204061415908b00f54a8b11f4c514ee#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R106), and provided upgrade notes in commit message (to be copied to release notes).

> Also minor update, the PR's description is outdated

Thanks, updated.
